crates/bevy_core/src/time/timer.rs

```diff
@@ -34,8 +37,57 @@ impl Timer {
         }
     }
 
+    pub fn pause(&mut self) {
```

Comment:
lets add `#[inline]` to all of these getters / setters

Reply:
Sounds good